src/skmultiflow/anomaly_detection/half_space_trees.py

```diff
@@ -197,18 +197,19 @@ def predict(self, X):
                 # Ensemble is empty, all classes equal, default to zero
                 predictions.append(0)
             else:
-                # if prediction of this instance being anomaly is greater than the threshold defined,
-                # then this instance is classified as an anomaly.
+                # if prediction of this instance is greater than the threshold
+                # defined,then this instance is classified as an anomaly.
```

Comment:
We could add a space after the comma (pretty much an optional, cosmetic, change). What do you think?

Reply:
Definetly. Updated in cc37f898c0cb91ed786c23f9413fad11b9929592